lib/cluster-store.js

```diff
@@ -59,6 +59,7 @@ module.exports = function(connectOrSession) {
   function ClusterStore(options) {
     Store.call(this, options);
     this._collection = NativeStore.collection(COLLECTION_NAME);
+    this._collection.configure(options);
```

Comment:
I am not very happy about mixing session.Store and NativeStore.Collection options in the same object.

I am proposing one of the following:

 - move collection config to `options.config`
 - or expose a new method `ClusterStore.prototype.configureCollection` as a wrapper for `this._collection.configure`

Thoughts?